### ISSUE TITLE
this track is not available

### ISSUE THREAD

**Issue by unkerjay** [mps-youtube/mps-youtube#588]
Doors, Leonard Cohen, Adele, Lady Gaga.

Most recent songs of Adele and Lady Gaga that PLAY in youtube are "not available".

I don't think this is a software bug as much as a deliberate block of said artists works.

"Happy Birthday" plays.

Maybe it's a software config issue.  Maybe it's a block.

**Comment by ids1024** [comment]
Could you post links to the specific videos, and the exact message you get?

**Comment by unkerjay** [comment]
I've attached images of the specific searches:

Adele
Lady Gaga
Isley Brothers
Frank Sinatra
Rudy Vallee (worked)
Ray Charles (worked)

On EACH I just picked the first video and got the same error message as
shown:

"Sorry this track is not available"

Now with Adele and Lady Gaga, they're likely they're most recent hits so
that they
would not be available is unlikely.

Course it didn't matter for them.  Picking ANY track of theirs got the same
response.

Certainly if it were a uniform problem with the software neither Rudy
Vallee or Ray
Charles would've worked either.  They did.

Hope this helps.

Just for fun I took the tracks that didn't work in mpsyt and went straight
to youtube,
same song.  Found it, played it, no problem.

Again, it didn't matter which song of Adele's or Lady Gaga's I picked.  The
results were
identical.

It wasn't this song worked, that one didn't.  NONE worked.  But, I could
find them and
play them in Youtube.

I even tried searching for some songs by title just to narrow it down.

Same problems.

**Comment by ids1024** [comment]
The images aren't showing. I think GitHub strips attachments in email replies.

It's odd that it would work in the browser but not mpsyt; what is the full output of `mpsyt --version`?

**Comment by unkerjay** [comment]
I think it's the latest:

0.2.7.1

just installed it after rebuilding my computer.

Who knew that so many programs were python dependent.

Several versions of how to install.

I went down several wrong paths and fried my system.

Good news, backup of the main system folders, Home directory on
it's own partition.  So, LOT of work to get back up to speed, but, nothing
critical lost.



On Sun, Feb 5, 2017 at 10:13 AM, Ian Douglas Scott <notifications@github.com

**Comment by ids1024** [comment]
Yes, that's the latest version. What does `mpsyt --version` list as the pafy version and backend? If it lists `internal` as the backend, that may cause issues (if so, try installing youtube-dl using pip3).

**Comment by UmutAlihan** [comment]
Yeap I had the same issue. It is solved by installing youtube-dl!!

**Comment by unkerjay** [comment]
I had youtube-dl installed already so I didn't re(install) it using the
method presented on github.

Installing it via that method does seem to have resolved the issue.

I picked "Stevie Wonder" and now I've got it on autoplay using "*".

I also tried it with Lady Gaga's "Million Reasons".

That worked too.

So, note, if you've got youtube-dl installed and songs aren't playing, one
thing to try is
the pip3 youtube-dl install.

So far so good.

Thanx

**Comment by unkerjay** [comment]
Had a small problem with it running just using "mpsyt".

Thought it might be an administrative problem.

So, I tried "sudo mpsyt".

That worked for me.

**Comment by ids1024** [comment]
> I had youtube-dl installed already so I didn't re(install) it using the
method presented on github.

Yeah, it needs to be the Python 3 version of youtube-dl, so mpsyt (which is Python 3) can call it as a library. The youtube-dl package on Arch Linux is Python 3, but the Debian/Ubuntu package is Python 2, and I don't know about others.

**Comment by lduktus** [comment]
I had also problems with some videos. It was kind of random which videos played and which not, but like 50% didn't worked. For me UmutAlihan's solution worked well. I installed youtube-dl and until now it seems to work well. I am working with Debian Testing and installed youtube-dl from the repositories, so for some people this could be a solution.

**Comment by RoastVeg** [comment]
This appears to be a problem with videos in the "webm 160k" format - 120k videos playback corretly even without installing youtube-dl with pip3 on Ubuntu.
